tests/phpunit/integration/Modules/Reader_Revenue_ManagerTest.php

```diff
@@ -744,7 +745,7 @@ public function test_feature_disabled_product_id_setting_not_registered() {
 		$this->assertFalse( $registered );
 	}
 
-	public function test_block_editor_script_enqueued() {
+	public function test_block_editor_assets_set_up() {
```

Comment:
Strictly speaking, these tests aren't asserting the actual enqueuing of the assets, so I've renamed them.